source/viz.md

```diff
@@ -1792,8 +1926,8 @@ Zoomed in `faithful`, raster (PNG, left) and vector (SVG, right) formats.
 
 ## Exercises
 
-Practice exercises for the material covered in this chapter 
-can be found in the accompanying 
+Practice exercises for the material covered in this chapter
+can be found in the accompanying
 [worksheets repository](https://github.com/UBC-DSCI/data-science-a-first-intro-worksheets#readme)
```

Comment:
This link is for the R version -- needs to be fixed to be the python repo

See #56